deploy/kubernetes/examples/demo-secret.yaml

```diff
@@ -8,4 +8,4 @@ stringData:
   management_address: demo-management-address  # Array management addresses
   username: demo-username                      # Array username
 data:
-  password: ZGVtby1wYXNzd29yZA==               # Array password
+  password: ZGVtby1wYXNzd29yZA==               # Array password - demo-password in base64
```

Comment:
how about `# base64 encoded Array password`? @rivkap what do you think?

Reply:
does it need "encoded"? if not the following is fine. either way.

# base64 array password

@oriyarde  @lisyukanna

Reply:
`base64 array password` sounds good to me

Reply:
done